miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/TagUtils.java

```diff
@@ -15,4 +18,30 @@ public static boolean instanceOf(Object o, String className) {
     }
   }
 
+  public static boolean isCurrentUser(String username) {
+    return username.toLowerCase()
+        .equals(getAuthentication().getName().toLowerCase());
+  }
+
+  public static boolean isAdmin() {
+    return hasRole("ROLE_ADMIN");
+  }
+
+  public static boolean isTech() {
+    return hasRole("ROLE_TECH");
+  }
+
+  private static boolean hasRole(String role) {
+    for (GrantedAuthority authority : getAuthentication().getAuthorities()) {
+      if (authority.getAuthority().toUpperCase().equals(role)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace these toUpperCase()/toLowerCase() and equals() calls with a single equalsIgnoreCase() call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1157)